certified-connectors/Delinea Secret Server/apiDefinition.swagger.json

```diff
@@ -0,0 +1,741 @@
+          "format": "int32"
+        },
+        "fieldName": {
+          "description": "The display name of the secret field.",
+          "type": "string"
+        },
+        "fileAttachmentId": {
+          "description": "If the field is a file attachment field, the id of the file attachment.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "filename": {
+          "description": "If the field is a file attachment field, the name of the attached file.",
+          "type": "string"
+        },
+        "isFile": {
+          "description": "Whether the field is a file attachment.",
+          "type": "boolean"
+        },
+        "isList": {
+          "description": "Whether or not the secret field is a list.",
+          "type": "boolean"
+        },
+        "isNotes": {
+          "description": "Whether the field is represented as a multi-line text box. Used for long-form text fields.",
+          "type": "boolean"
+        },
+        "isPassword": {
+          "description": "Whether the field is a password. Password fields are hidden by default in the UI and their value is not returned in GET calls that return secrets. To retrieve a password field value, make a GET call to /api/secrets/{secretId}/fields/{slug}.",
+          "type": "boolean"
+        },
+        "itemId": {
+          "description": "The id of the secret field item. Leave empty when creating a new secret.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "itemValue": {
+          "description": "The value of the secret field item. For list fields, this is a comma-delimited list of the list id guids that are assigned to this field.",
+          "type": "string"
+        },
+        "listType": {
+          "$ref": "#/definitions/SecretFieldListType"
+        },
+        "slug": {
+          "description": "A unique name for the secret field on the template. Slugs cannot contain spaces and are used in many places to easily refer to a secret field without having to know the field id.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "HeartbeatStatus": {
+      "description": "Current status of heartbeat",
+      "properties": {},
+      "type": "string",
+      "enum": [
+        "Failed",
+        "Success",
+        "Pending",
+        "Disabled",
+        "UnableToConnect",
+        "UnknownError",
+        "IncompatibleHost",
+        "AccountLockedOut",
+        "DnsMismatch",
+        "UnableToValidateServerPublicKey",
+        "Processing",
+        "ArgumentError",
+        "AccessDenied"
+      ]
+    },
+    "SecretFieldListType": {
+      "description": "The type of list. Valid values are “None”, “Generic”, and “URL”.",
+      "properties": {},
+      "type": "string",
+      "enum": [
+        "Generic",
+        "URL",
+        "None"
+      ]
+    },
+    "SecretTemplateModel": {
+      "description": "SecretTemplateModel[]",
```

Comment:
@tylerezimmerman descriptions should be valid sentences. You can refer [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions) and [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).